lib/Search/Provider.php

```diff
@@ -103,14 +103,17 @@ public function search(IUser $user, ISearchQuery $query): SearchResult {
 				}
 
 				return new SearchResultEntry(
-					'',
+					$this->urlGenerator->linkToRoute('mail.avatars.image', [
+						'email' => $message->getFrom()->first()->getEmail(),
```

Comment:
1) There might not be a sender
2) The sender might not have an email

Does not makes sense, I know, but there are systems where this happens for unknown reasons …

Reply:
Yes, I'll add error handling :-)

Reply:
done